### PR TITLE
add documentation on how to manage menu trees

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: book/navigation.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+the translation domain using the ``translateDomain`` option (by default, the translator's default
+domain will be used).
+
+.. _book-navigation-compose-tree:
+
+Organize the Navigation Trees
+-----------------------------
+
+The next step is to compose a tree off of the menu items that you created before. Theses trees are
+build under the ``tree`` key:
```
Comment (akuzmenko-magecore) [inline book/navigation.rst]: - "compose a tree ~~off~~ of..." ?
- "These~~s~~ trees are..."

Comment (akuzmenko-magecore) [inline book/navigation.rst]: Closing this PR in favor of https://github.com/orocrm/documentation/pull/158.
It includes the review fixes and merge with master to resolve conflicts.

Comment (xabbuh) [inline book/navigation.rst]: Thank you Alexander. Unfortunately, I somehow missed your comment.

File: book/navigation.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            blog_index:
+                label: acme_demo.menu.blog_overview
+                route: acme_demo.blog_index
+
+The example above defines three menu items:
+
+* The ``blog`` item consists of a label and the URI ``#``. This means that the item will not react
+  on mouse clicks, but can be used as a placeholder for nested menus.
+
+* Both the ``blog_categories`` and the ``blog_index`` items each reference a an existing route.
```
Comment (nnenasheva) [inline book/navigation.rst]: Should be  

Both the ``blog_categories`` and the ``blog_index`` items reference an existing route. 

or 

Each of the ``blog_categories`` and the ``blog_index`` items references an existing route.

File: book/navigation.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  Thus, when the user later clicks on of these items, they will enter a page that is rendered by
```
Comment (nnenasheva) [inline book/navigation.rst]: clicks one of these items, they will get to a page

File: book/navigation.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+  the controller that is responsible for the configured route.
+
+As you can see, the menu item labels will be translated by default. Hence you can use arbitrary
+labels here as long as they can be translated by configured ``translator`` service. You can change
```
Comment (nnenasheva) [inline book/navigation.rst]: , as long as

File: book/navigation.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+the translation domain using the ``translateDomain`` option (by default, the translator's default
+domain will be used).
+
+.. _book-navigation-compose-tree:
+
+Organize the Navigation Trees
+-----------------------------
+
+The next step is to compose a tree off of the menu items that you created before. Theses trees are
```
Comment (nnenasheva) [inline book/navigation.rst]: tree of

**Pull request closed without merging**